src/components/AllBreweriesHome/AllBreweriesHome.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
+import BreweryInterface from "../BreweryCard/BreweryCard";
+
+export default function AllBrieweriesHome() {
+    const [breweries, setBreweries] = useState([]);
+    useEffect(() => {
+        fetch('https://api.openbrewerydb.org/v1/breweries?fbclid=IwZXh0bgNhZW0CMTAAAR1QKdPcqOwBI51QmYcnitDSFfKOnHlOLDSbxXadLJZgJJWfQ797QPTltHA_aem_ZmFrZWR1bW15MTZieXRlcw')
```

Comment:
You can export the request in a separate folder (api for example). It is good practice to keep the requests in a single folder location and not scatter them all around the app

Reply:
i think that i understood you and made a new folder named "api", then i moved the file and it's corresponding css file there in  a separate folder, from now on all files that include fetch will be found there